elements/urth-viz-chart/urth-viz-chart.html

```diff
@@ -90,6 +90,15 @@
                 notify: true,
                 observer: '_typeChanged'
             },
+
+            /**
+             * Allows user to select multiple chart components (bar in barchart, slice in pie chart, etc).
+             */
+            multiSelect: {
```

Comment:
No need for property here once it is moved to `elements/urth-viz-chart/urth-viz-chart-common-behavior.html`.